Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.yml

```diff
@@ -113,6 +119,17 @@ args:
   - weighted avgm
   required: false
   secret: false
+  deprecated: true
+- default: false
+  defaultValue: 'true'
+  description: Whether to extract keyworks for model.
```

Comment:
Whether to extract keywords for the model. Can be "true" or "false". Default is "true".